src/witan/models/dem/ccm/fert/fertility.clj

```diff
@@ -224,31 +231,13 @@
   [inputs params]
   (project-asfr-internal inputs params))
 
-(defworkflowfn project-births-1-0-0
-  "Project births from fixed rates. Takes a dataset with population at risk 
-   from the current year of the projection loop and another dataset with 
-   fixed fertility rates for the population. Returns a dataset with a column 
-   of births, which are the product of popn at risk & the rates"
-  {:witan/name :ccm-fert/project-births
-   :witan/version "1.0.0"
-   :witan/input-schema {:initial-projected-fertility-rates ProjFixedASFRSchema
-                        :population-at-risk PopulationAtRiskSchema}
-   :witan/output-schema {:births-by-age-sex-mother BirthsAgeSexMotherSchema}
-   :witan/exported? true}
-  [{:keys [initial-projected-fertility-rates population-at-risk]} _]
-  (let [projected-births (cf/project-component-fixed-rates
-                          population-at-risk
-                          initial-projected-fertility-rates
-                          :fert-rate :births)]
-    {:births-by-age-sex-mother projected-births}))
-
-(defworkflowfn project-births-1-1-0
+(defworkflowfn project-births
```

Comment:
agreed with @acron0 put the version number back in - where we need to change these fns we should add version numbers